src/styles/auto-prefix.js

```diff
@@ -3,6 +3,15 @@ import warning from 'warning';
 
 const prefixers = {};
 
+const pruneNulls = style => {
+  Object.keys(style).forEach(key => {
+    if (style[key] === null || style[key] === undefined) {
+      delete style[key];
```

Comment:
I'm not 100% sure, but I have seen that using `delete` is bad for performances reason. In chrome, it's breaking the hidden class of the object and require more work at the assembly level.